### PR TITLE
User can see articles on mobile homescreen (api setup)

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description
Changes proposed in this pull request:
* add api namespace with articles controller
* setup routes for api articles index
* setup api endpoint for article index

#### What I have learned working on this feature:
* how to setup an api endpoint

### CODE REVIEW

File: app/controllers/api/v1/articles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Api::V1::ArticlesController < ApplicationController
+	def index
+		@articles = Article.all
```
Comment (oliverochman) [inline app/controllers/api/v1/articles_controller.rb]: do you need to use an instance variable here?

Comment (LauraRe) [reply app/controllers/api/v1/articles_controller.rb]: removed

File: spec/requests/api/v1/articles_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+RSpec.describe Api::V1::ArticlesController, type: :request do
+	describe 'GET /api/v1/articles' do
+		before do
+			category = create(:category)
+			5.times { create( :article, category: category ) }
+		end
+		it 'returns a collection of articles' do
+			get '/api/v1/articles'
+			expect(JSON.parse(response.body)['articles'].count).to eq 5
```
Comment (oliverochman) [inline spec/requests/api/v1/articles_spec.rb]: extract `JSON.parse(response.body)` to a helper method

Comment (LauraRe) [reply spec/requests/api/v1/articles_spec.rb]: done

**Pull request merged**